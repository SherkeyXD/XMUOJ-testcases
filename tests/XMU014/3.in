27
716423815537161633334323751742564417878225617226
22
17861381238484637843585631886268582651352
17
3616372575785627656241256248367266264485263
26
7333666852726852213663367336354323636113
12
364462167438737647548538655687677867666
4
83273855663673364318875386818788436466533336615
18
214361286886532657838
35
782346158376778882133646331813168758
24
667426271353853766273111655147466351152
35
67215746848637331147334286655511715554
15
3383446158263436763431834666711283
1
45333
20
7464377676376255646713862133644
0
7736763643615586268
