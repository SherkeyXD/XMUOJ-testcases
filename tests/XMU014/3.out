658
765
2451
456
7453
10070406328
120
177
469
187
1138
378
419
7736763643615586268
